validation-testing/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.brooklyncentral.catalog</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>validation-testing</artifactId>
+
+    <name>Brooklyn Community Catalog Blueprint Validation Testing</name>
+    <description>
+       Tests to determine if blueprint repositories are correctly formatted
+       and can be successfully scraped by the catalog server.
+    </description>
+
+    <properties>
+        <testng.version>6.1.1</testng.version>
+        <jgit.version>4.3.0.201604071810-r</jgit.version>
+        <guava.version>19.0</guava.version>
+        <commons-io.version>2.4</commons-io.version>
+        <util-validator.version>1.0-SNAPSHOT</util-validator.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.jgit</groupId>
+            <artifactId>org.eclipse.jgit</artifactId>
+            <version>${jgit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>${commons-io.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.cloudsoft.catalog</groupId>
+            <artifactId>util-validator</artifactId>
+            <version>${util-validator.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```

Comment:
You will need to add the artifactory repository to get the `util-validator` module
```xml
    <repositories>
        <repository>
          <snapshots>
            <enabled>false</enabled>
          </snapshots>
          <id>cloudsoft-release</id>
          <name>libs-release</name>
          <url>https://artifactory.cloudsoftcorp.com/artifactory/libs-release</url>
        </repository>
        <repository>
          <snapshots />
          <id>cloudsoft-snapshots</id>
          <name>libs-snapshot</name>
          <url>https://artifactory.cloudsoftcorp.com/artifactory/libs-snapshot</url>
        </repository>
      </repositories>
```